splunk_connect_for_snmp/snmp/manager.py

```diff
@@ -294,7 +294,7 @@ def do_work(
         self,
         ir: InventoryRecord,
         walk: bool = False,
-        profiles: List[str] = None,
+        profiles: List[str] = [],
```

Comment:
I not sure that empty list as default value good idea:
* https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil
* https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python


Can we use `profiles: List[str] = None` and add to body function:


```
if profiles is None:
   ...
```

Reply:
changed